script/plugin_tools.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -ev
+
+if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then
+  export FLUTTER_CHANGED_PACKAGES=""
+else
+  export FLUTTER_CHANGED_PACKAGES=`git diff --name-only $TRAVIS_COMMIT_RANGE | grep -o "packages/[^/]*" | sed -e "s/packages\///g" | sort | uniq | paste -s -d, -`
```

Comment:
(not so uncommon) situation: I enable a new lint in the analyzer options. I fix it in all plugins, but I miss one (easy to do with the number of plugins we have, happened to me before). Travis on the PR is now green because it only runs on the plugins that changed. Travis then turns red post-submit. @goderbauer is sad.